src/frontend/citizen-portal/src/app/components/dispute-ticket-summary/dispute-ticket-summary.component.html

```diff
@@ -17,38 +17,55 @@ <h2><strong> Disputant contact information</strong></h2>
             <div class="text">{{ noticeOfDispute.disputant_given_names }}</div>
           </div>
         </div>
-
+        <h2><strong> Contact information</strong></h2>
         <div class="row mb-2">
+          <div class="col-lg-4">
+            <small class="d-block">Contact type</small>
+            <div class="text">{{ noticeOfDispute.contact_type | titlecase }}</div>
+          </div>
           <div class="lg-2 col-lg-4">
-            <small class="d-block">Mailing address</small>
-            <div class="text">{{ noticeOfDispute.address }}</div>
+            <small class="d-block" *ngIf="noticeOfDispute.contact_type === ContactType.Lawyer">Law firm</small>
+            <div class="text" *ngIf="noticeOfDispute.contact_type === ContactType.Lawyer">{{ noticeOfDispute.contact_law_firm_name }}</div>
           </div>
-          <div class="col-lg-4">
-            <small class="d-block">Country</small>
-            <div class="text">{{ getCountryLongName(noticeOfDispute.address_country_id) }}</div>
+        </div>
+        <div class="row mb-2">
+          <div class="lg-2 col-lg-4">
+            <small class="d-block">{{ noticeOfDispute.contact_type === ContactType.Individual ? 'S' : (noticeOfDispute.contact_type === ContactType.Lawyer ? 'Lawyer s' : 'Other s') }}urname</small>
```

Comment:
Prefer using ngIf instead of concat the string inside for this case, try to keep the keyword `surname` complete.
At least split by `===ContactType.Individual` and `!=ContactType.Individual` -> Lawyer/Other